README.md

```diff
@@ -1,63 +1,122 @@
+Current macOS version: *Sierra (10.12)*, tested with VirtualBox versions *5.1.6r110634* and *5.1.8r111374*.
 
-To build a VM running macOS, follow the directions below:
+To build a VM running macOS, follow the directions below.
 
-  1. Download the installer from Mac App Store (it should be available in the 'Purchases' section if you've acquired it previously). The installer will be placed in your Applications folder.  (Should work for Yosemite, El Capitan and Sierra - 10.10-10.12.)
-  2. Make the script executable and run it: `chmod +x prepare-iso.sh && ./prepare-iso.sh`.
+1.  Download the installer from Mac App Store (it should be available in the 'Purchases' section if you've acquired it
+    previously). The installer will be placed in your Applications folder. (Should work for Yosemite, El Capitan and
+    Sierra - 10.10-10.12.)
 
-      If the script fails to find the installer you can specify its path as the first parameter. By default, the output is saved as <Yosemite|El Capitan|Sierra>.iso on the Desktop. You can change this using the second parameter.
-      Example:
+2.  Run `./prepare-iso.sh`.
 
-        ```./prepare-iso.sh /Applications/Install\ macOS Sierra\ 2.1\ Beta\ 2.app /Users/Steve/sierra-2.1-b2.iso```
+    If the script fails to find the installer you can specify its path as the first parameter. By default, the output
+    is saved as <Yosemite|El Capitan|Sierra>.iso on the Desktop. You can change this using the second parameter. For
+    example:
 
-  3. Open VirtualBox and create a new VM.
-  4. Set:
-      - name: your choice
-      - type: `OS X`
-      - version: `Mac OS X 10.11 El Capitan (64-bit)` or `macOS Sierra`, if your version of VirtualBox has been updated to include it.
-  5. Follow the rest of the VM creation wizard and either leave the defaults or adjust to your liking. You may want to increase the video memory from the VirtualBox default of 16MB to 64MB or 128MB. While the current release of macOS Sierra does boot with just 16MB, interace performance is severly constraint.
-  6. In Terminal, run the command `VBoxManage modifyvm "{vmname}" --cpuidset 00000001 000306a9 00020800 80000201 178bfbff` (where `"{vmname}"` is the exact name of the VM set in step 4) so the VM has the right CPU settings for macOS.
-  7. To prevent choppiness in the VM, go into settings and uncheck the 'Enable Audio' option under 'Audio'.
-  8. Click 'Start' to boot the new VM.
-  9. Select the iso created in step 2 when VirtualBox asks for it.
-  10. In the installer, select your preferred language.
-  11. Go to `Utilities > Disk Utility`. Select the VirtualBox disk and choose `Erase` to format it as a `Mac OS Extended (Journaled)` drive.
-  12. Quit Disk Utility, and then continue with installation as normal.
+    ```shell
+    ./prepare-iso.sh \
+      /Applications/Install\ macOS Sierra\ 2.1\ Beta\ 2.app \
+      /Users/Steve/sierra-2.1-b2.iso
+    ```
 
+3.  Open VirtualBox and create a new VM.
```

Comment:
Since the intention of this readme is display on GitHub (or elsewhere as rendered markdown), I'd rather not add spaces between the list items—especially since in markdown syntax it's generally better to have ol/ul on subsequent lines (with no extra gap).